### DISCUSSION TITLE
(WiinUSoft) Controller no longer connecting to computer

### DISCUSSION THREAD

**Discussion by redpuyo24** [KeyPuncher/WiinUPro#187] [Help]
WiinUSoft (latest version) was working fine for my PC (Windows 11) with my Wii U Pro Controller until a few months ago, when I reconnected it to the WiiU, and now it won't connect to my computer. I force disconnected it from my WiiU using the "delete all controller pairings" setting, but that still didn't work. It won't even show up in my computer's bluetooth menu and trying to pair with WiinUSoft shows "Searching for controllers..." and never finding it. Is it possible for me to reconnect the Wii U Pro Controller or am I just going to have to get a different controller? 

(I've tried this on a different computer and it still doesn't work, and I can connect other bluetooth devices to this computer just fine, so it's the controller that's giving me issues.)

**Comment by KeyPuncher** [comment]
This is the first time I've heard of this particular case of a controller becoming no longer discoverable. I've seen connecting to a Wii U solve problems, not create them so I find this very unusually but I have no helpful information unfortunately.
